Cargo.lock

```diff
@@ -6543,13 +6555,13 @@ dependencies = [
  "sp-version",
  "sp-wasm-interface",
  "tracing",
- "wasmi",
+ "wasmi 0.9.1",
```

Comment:
Still 0.9.1 appears in substrate
Seems that merged substrate patch missed it

Reply:
fixed